src/vis/vis.js

```diff
@@ -147,8 +147,9 @@ var VisModel = Backbone.Model.extend({
       windshaftSettings: windshaftSettings
     });
 
+    var allowScrollInOptions = (vizjson.options && vizjson.options.scrollwheel) || vizjson.scrollwheel;
```

Comment:
I think we can safely ignore `vizjson.scrollwheel` now that we now it's not part of the viz.json.

Reply:
If we ignore them several tests will fail...

Reply:
We should fix them IMO.

Reply:
agreed but is out of the scope of this issue

Reply:
Then we should leave this as it was and open an issue to fix the bug. Otherwise we might never get back to it and change will remain outdated.